src/pages/NIHicWebform.js

```diff
@@ -0,0 +1,2266 @@
+                        'Study Type',
+                      ]),
+                    ]),
+                  div(
+                    {className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'},
+                    [
+                      h(Select, {
+                        name: 'studyType',
+                        id: 'studyType',
+                        onChange: () => {},
+                        blurInputOnSelect: true,
+                        openMenuOnFocus: true,
+                        isDisabled: false,
+                        isClearable: true,
+                        isMulti: false,
+                        isSearchable: true,
+                        options: [],
+                        placeholder: 'Select a study type...',
+                        className: '',
+                        required: true,
+                      })
+                    ])
+                ]),
+
+                div({className: 'form-group'}, [
+                  div(
+                    {className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'},
+                    [
+                      label({className: 'control-label rp-title-question common-color'}, [
+                        'Project title for data to be submitted',
+                      ]),
+                    ]),
+                  div(
+                    {className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group rp-last-group'},
+                    [
+                      input({
+                        type: 'text',
+                        name: 'projectTitle',
+                        id: 'projectTitle',
+                        maxLength: '256',
+                        value: '',
+                        onChange: () => {},
+                        className: 'form-control',
+                        required: true,
+                      })
+                    ])
+                ]),
+
+                div({className: 'form-group'}, [
+                  div(
+                    {className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'},
+                    [
+                      label({className: 'control-label rp-title-question common-color'}, [
+                        'Is this a multi-center study?',
+                      ]),
+                    ]),
+                  div({ className: 'row no-margin' }, [
+                    div({ className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 dataset-group' }, [
+
+                      RadioButton({
+                        style: {
+                          margin: '2rem',
+                          color: ' #010101',
+                        },
+                        id: 'checkPublicAccess_yes',
+                        name: 'checkPublicAccess',
+                        value: 'yes',
+                        defaultChecked: publicAccess,
+                        onClick: () => this.setPublicAccess(true),
```

Comment:
The original code where this came from was for updating a dataset. This radio button really doesn't have anything to do with that. There probably shouldn't be any handler on this, I think it might work with just
```suggestion
                        onClick: () => {},
```